vagrant_box_defaults.rb

```diff
@@ -5,7 +5,7 @@
 $SERVER_BOX = "cilium/ubuntu-dev"
 $SERVER_VERSION= "172"
 $NETNEXT_SERVER_BOX= "cilium/ubuntu-next"
-$NETNEXT_SERVER_VERSION= "54"
+$NETNEXT_SERVER_VERSION= "55"
```

Comment:
AFAIK we need to bump it from a separate PR which branch has a special name (to populate VM cache).